Tutorials/intro-to-qdac2/Integration_with_the_OPX_and_pyvisa.ipynb

```diff
@@ -450,11 +471,34 @@
     "qdac.write(f\"sour1:volt {0}\")\n",
     "qdac.write(f\"sour2:volt {0}\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Example:\n",
+    "1. Setting the QDAC on a specific voltage - in the sexample you can set the QDAC on a specific, fixed, value. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qdac2_driver import QDACII, set_QDAC_voltage\n",
+    "\n",
+    "set_QDAC_voltage(\n",
+    "    qdac,\n",
+    "    channel=1,\n",
+    "    voltage=0.0,\n",
```

Comment:
It will maybe be more readable in one line:
set_QDAC_voltage(qdac, channel=1, voltage=0.235)